src/services/role/model.js

```diff
@@ -36,7 +39,8 @@ roleSchema.statics.resolvePermissions = function resolvePermissions(roleIds) {
 	const permissions = new Set();
 
 	function resolveSubRoles(roleId) {
-		return roleModel.findById(roleId) // fixme
+		return roleModel
```

Comment:
Codacy found an issue: ['roleModel' was used before it was defined.](https://app.codacy.com/gh/hpi-schul-cloud/schulcloud-server/pullRequest?prid=6045788)